### ISSUE TITLE
is sdcard working for surface pro 6 with surface linux?

### ISSUE THREAD

**Issue by trcrsired** [linux-surface/linux-surface#1586]

**Comment by chrislogan2** [comment]
Per the current feature matrix, yes
https://github.com/linux-surface/linux-surface/wiki/Supported-Devices-and-Features#feature-matrix

**Comment by crazyfox-ua** [comment]
@trcrsired yes, it works on `6.10.10-surface-1` like a charm - right now working with 512Gb Samsung card installed
